cli/valet.php

```diff
@@ -165,8 +165,7 @@
     /**
      * Stop serving the given domain over HTTPS and remove the trusted TLS certificate.
      */
-    $app->command('unsecure [domain] [--all]', function ($domain = null, $all) {
-
+    $app->command('unsecure [domain] [--all]', function ($domain = null, $all = false) {
```

Comment:
It looks like symfony prefers `null` here instead of `false`

I know this was trying to fix:
```
Deprecated: Required parameter $all follows optional parameter $domain 
in ~/.composer/vendor/laravel/valet/cli/valet.php on line 168
```

But using `false` triggers this:
```
Fatal error: Uncaught Symfony\Component\Console\Exception\LogicException: Cannot set a default value when using InputOption::VALUE_NONE mode. in ~/.composer/vendor/laravel/valet/vendor/symfony/console/Input/InputOption.php:159
Stack trace:
#0 ~/.composer/vendor/laravel/valet/vendor/mnapoli/silly/src/Command/Command.php(63): Symfony\Component\Console\Input\InputOption->setDefault(false)
#1 ~/.composer/vendor/laravel/valet/vendor/mnapoli/silly/src/Application.php(108): Silly\Command\Command->defaults(Array)
#2 ~/.composer/vendor/laravel/valet/cli/valet.php(181): Silly\Application->command('unsecure [domai...', Object(Closure))
#3 {main}
  thrown in ~/.composer/vendor/laravel/valet/vendor/symfony/console/Input/InputOption.php on line 159
```

Reply:
Switching to `null` makes both these messages go away.

Reply:
Oops, looks like I missed this when running the testsuite somehow 😳

Reply:
It might be a factor of my local environment having mixed PHP versions and mixed composer installs. Perhaps the test suite isn't triggering it because it's actually installing only a single specific compatible PHP version each time.

Feel free to PR the update 👍

Reply:
Opened #988